tests/integration/test_cloud_storage_api_client.py

```diff
@@ -0,0 +1,98 @@
+import os
+import pytest
+
+from datetime import datetime
+
+from mozmlops.cloud_storage_api_client import CloudStorageAPIClient
+
+def test_store_fetch_delete__nominal():
+    """
+    Test the integration to store a file, fetch a file, and delete a file. 
+    This test takes several seconds to run. Exclude integration tests except before release with the command:
+    `pytest --ignore=tests/integration`
+
+    If this test fails, one of the integrations is broken. 
+    Each has its own assertions and error messages, designed to help pinpoint the failure.
+    You can also visit the bucket on GCS to find the file:
+
+    https://console.cloud.google.com/storage/browser/mozdata-tmp
+
+    These functions are tested together as a top-level integration test because: 
+    
+    1. .store() and .fetch() are inverse operations, and then .delete() cleans up after .store()
+    2. Testing these functions separately requires independently recreating the integrations,
+    3. Which duplicates the logic while failing to actually isolate the integration behaviors.
+    """
+    # Given an artifact store and a file containing Ada Lovelace's name:
+
+    storage_client = CloudStorageAPIClient(project_name="mozdata", bucket_name="mozdata-tmp")
+
+    string_to_store = "Ada Lovelace"
+    identifier = os.getlogin() if os.getlogin() else "anonymous"
```

Comment:
We could indeed. I chose `os.getlogin()` because it seemed like it would be a nice developer experience for someone running tests to easily identify the test files they had generated because their name is on them—with "anonymous" as the backup for systems that don't provide this. But I'm not married to the idea. Happy to change over if you'd prefer UUID.

Reply:
Yeah, I think we might want to get rid of this, unless we add some more error catching boilerplate around it: the call is not returning `None` on my system, something weird is instead happening with exceptions. Given that we don't care too much about this specific function, I'd recommend getting rid of it entirely. We already add a timestamp to the filename, so we might not even need to bring in UUIDs as I first suggested!

Reply:
Sounds good to me! I'll remove. It's _theoretically possible_ that two different MLOps engineers run the test at the _exact same second_ and get the same filename here...but it's pretty unlikely.